Compass129Sources/COMfile/C12COM9.ASC

```diff
@@ -814,35 +814,35 @@ m_chk_tpa	ld	hl,tab_TPA	;check for page 0
 	ret
 
 m_manage	ld	ix,tempbuffer
-	call	m_alloc_tpa	;alloceer 2 tpaslotsegm. naar tempbuffer
+	call	m_alloc_tpa	;allocate 2 tpaslotsegm. to temp buffer
 	ret	c
 	call	m_alloc_tpa
-	;de eerste dient voor cblock1
-	;de tweede mag ontbreken, deze dient voor de labelbuffer
-	;als hij ontbreekt, wordt hij later in de maintpa-segm. gezet
-	;Merk op: de laatste 2 bytes van tempbuffer zullen dan 0 zijn
-	;hier wordt rekening mee gehouden bij m_add_temp
-	ld	ix,buffer	;merkop:2 tpa gevonden,dan ix al goed
+	;the first is for cblock1
+	;the second may be missing, it is for the label buffer
+	;if it's missing it'll be in the maintpa segm later. set

```

Comment:
'gezet' is part of 'wordt ... gezet' ('... will be set'), so 'set' shouldn't be at the end as part of a new sentence.
Instead, try: "it'll be *set* in the maintpa-segment later."